src/general/MainMenu.cs

```diff
@@ -752,6 +790,33 @@ private void WarnAboutNoEnabledMods()
         }
     }
 
+    private float TrackMenuPerformance(float delta)
+    {
+        var currentFrameRate = 1 / delta;
```

Comment:
I think the framerate should be read from Godot, there's an engine method to get the current framerate. Also I think the performance measurement should only begin one second in to the menu so that an initial lag spike is not counted. Also just to confirm that the measuring shouldn't happen before the intro video if it is enabled ends.